lib/Parser.js

```diff
@@ -1732,9 +1744,8 @@ class Parser extends Tapable {
 
 		const arr = [];
 		if(expression.elements)
-			expression.elements.forEach(expr => {
```

Comment:
parseStringArray and parseCalculatedStringArray is also unused, could you remove them too?